target/linux/ramips/image/mt7621.mk

```diff
@@ -118,6 +118,20 @@ define Device/pbr-m1
 endef
 TARGET_DEVICES += pbr-m1
 
+define Device/r6220
+  DTS := R6220
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  KERNEL_SIZE := 4194304
+  IMAGE_SIZE := $(ralink_default_fw_size_32M)
```

Comment:
Your factory or kernel + ubi partition in the DTS doesn't match the size defined in the ```ralink_default_fw_size_32M``` variable.